README.md

```diff
@@ -188,6 +188,15 @@ To package a model, check out [model archiver documentation](model-archiver/READ
 
 Browse over to the [Docs readme](docs/README.md) for the full index of documentation. This includes more examples, how to customize the API service, API endpoint details, and more.
 
+## External demos powered by MMS
```

Comment:
Looks great. Thanks for adding this :) Could you also add a reference to this in the table-of-contents.